### ISSUE TITLE
No collection found on Google Workspace calendars - is there any way to sync a specific CalDAV calendar to a specific Google calendar?

### ISSUE THREAD

**Issue by tromlet** [pimutils/vdirsyncer#1079]
Before you submit bug reports: https://vdirsyncer.pimutils.org/en/stable/contributing.html

Things to include in your bugreport:

| Question | Response |
|---|---|
| **Your vdirsyncer version** | `0.19.1` |
| **If applicable, which server software (and which version) you're using** | ? |
| **Your Python version** | Python 3.10.10 |
| **Your operating system** | Alma Linux 8.7 |
| **Your config file** | https://privatebin.net/?d900aa68dba5b714#8wbzSmF6iUFFkBmLxPbJmbMpBKthSpmSR4jcBD6rMFrM - Password: `bunbunCARROT33!` |

Use `vdirsyncer -vdebug` for debug output. The output is sensitive, but please attach at least the last few lines before the error (if applicable), censored as necessary. This is almost always the most useful information.

This is the standard output from a `vdirsyncer discover`, but there's a link to the full and sanitized version (with password) down below.

```shell
$ vdirsyncer discover
Discovering collections for pair example
baikal_example_local:
  - "default" ("Example")
google_example_remote:
  - "events" ("exampleadmin@gmail.com")
warning: No collection "default" found for storage google_example_remote.
Should vdirsyncer attempt to create it? [y/N]: y
error: The server returned something vdirsyncer doesn't understand. Error message: InvalidXMLResponse("Couldn't find home-set.")
error: While this is most likely a serverside problem, the vdirsyncer devs are generally interested in such bugs. Please report it in the issue tracker at https://github.com/pimutils/vdirsyncer/issues
```

https://privatebin.net/?48cc4691573cc2d5#6PQ4TBiAbit9kCoZidDQAJj83gcPxkWYSeezoU8goMNK

Password: `catsRneat42!`

Basically, it doesn't seem like there's a way to specify WHICH calendar on Google to synchronize to - and I get this error regardless of whether or not I try to synchronize Google via the `type = "caldav"` or `type = "google_calendar"` in my config file.

**Comment by surfingtomchen** [comment]
same here

**Comment by WhyNotHugo** [comment]
Pastebin links have expired:

> Could not get paste data: Paste does not exist, has expired or has been deleted.

To specify which collections need to be synchronised, use the `collections` configuration parameter:

https://vdirsyncer.pimutils.org/en/stable/config.html?highlight=collections#pair-section
